### PR TITLE
chore(rpc): implement input, a11y, console

### PR DESCRIPTION
**Pull request by pavelfeldman**

### CODE REVIEW

File: src/rpc/client/accessibility.ts [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+export class Accessibility {
+  private _channel: PageChannel;
+
+  constructor(channel: PageChannel) {
+    this._channel = channel;
+  }
+
+  snapshot(options: { interestingOnly?: boolean; root?: ElementHandle } = {}): Promise<types.SerializedAXNode | null> {
+    const root = options.root ? options.root._elementChannel : undefined;
+    return this._channel.accessibilitySnapshot({ options: { interestingOnly: options.interestingOnly, root } });
```
Comment (dgozman) [inline src/rpc/client/accessibility.ts:31]: Shouldn't the recursive handle->guid unwrapper handle the `root`?

File: src/rpc/connection.ts [hunk 2]
```diff
@@ -110,6 +114,8 @@ export class Connection {
           return obj.emit;
         if (prop === 'on')
           return obj.on;
+        if (prop === 'once')
+          return obj.on;
```
Comment (dgozman) [inline src/rpc/connection.ts:118]: obj.once

File: src/rpc/server/pageDispatcher.ts [hunk 3]
```diff
@@ -83,6 +86,9 @@ export class PageDispatcher extends Dispatcher implements PageChannel {
   }
 
   async waitForEvent(params: { event: string }): Promise<any> {
+    const result = await this._page.waitForEvent(params.event);
+    if (result instanceof ConsoleMessage)
```
Comment (dgozman) [inline src/rpc/server/pageDispatcher.ts:90]: Might as well do Request, Response and Frame.

**Review by dgozman — APPROVED** [review]

File: src/rpc/client/accessibility.ts (hunk 1 above)
Comment (pavelfeldman) [reply src/rpc/client/accessibility.ts:31]: I need to go from ElementHandle to ElementHandleChannel here, recursive wrapper goes Channel -> guid

File: src/rpc/connection.ts (hunk 2 above)
Comment (pavelfeldman) [reply src/rpc/connection.ts:118]: oops

File: src/rpc/server/pageDispatcher.ts (hunk 3 above)
Comment (pavelfeldman) [reply src/rpc/server/pageDispatcher.ts:90]: Done

**Pull request merged**